cmd/cloud_loadzone.go

```diff
@@ -5,14 +5,15 @@ import (
 	"go.k6.io/k6/cloudapi"
 )
 
-func getCloudLoadZoneCmd(client *cloudapi.K6CloudClient, c *cmdCloud) *cobra.Command {
+func getCloudLoadZoneCmd(client *cloudapi.K6CloudClient) *cobra.Command {
```

Comment:
I decoupled cloud commends from the `cmdCloud` object, as the latter is used for running tests